crates/neon/src/thread/mod.rs

```diff
@@ -0,0 +1,232 @@
+//! Thread-local storage for JavaScript threads.
+//!
+//! At runtime, an instance of a Node.js addon can contain its own local storage,
+//! which can then be shared and accessed as needed between Rust modules. This can
```

Comment:
I don't understand this bit, "as needed between Rust modules".

```suggestion
//! which can then be shared and accessed as needed from Rust in a Neon module. This can
```